scripts/pipeline/batch/submit.py

```diff
@@ -1,31 +1,40 @@
+import os
+
 import boto3
 
 batch = boto3.client("batch", region_name="us-east-1")
 
-NR_OF_TILES_IN_SAMPLE_FILE = 1517
+NR_OF_TILES_IN_SAMPLE_FILE = 2728
+
+PC_KEY = os.environ["PC_SDK_SUBSCRIPTION_KEY"]
 
-PC_KEY = "***"
 
-for i in range(NR_OF_TILES_IN_SAMPLE_FILE):
-    job = {
-        "jobName": f"fetch-and-run-{i}",
-        "jobQueue": "fetch-and-run",
-        "jobDefinition": "fetch-and-run",
-        "containerOverrides": {
-            "command": ["datacube.py", "--index", f"{i}", "--bucket", "clay-tiles-02"],
-            "environment": [
-                {"name": "BATCH_FILE_TYPE", "value": "zip"},
-                {
-                    "name": "BATCH_FILE_S3_URL",
-                    "value": "s3://clay-fetch-and-run-packages/batch-fetch-and-run.zip",
-                },
-                {"name": "PC_SDK_SUBSCRIPTION_KEY", "value": f"{PC_KEY}"},
-            ],
-            "resourceRequirements": [
-                {"type": "MEMORY", "value": "8000"},
-                {"type": "VCPU", "value": "4"},
-            ],
-        },
-    }
+job = {
```

Comment:
We are, but instead of running 2500 individual tasks, we are running one [array job](https://docs.aws.amazon.com/batch/latest/userguide/array_jobs.html) which bundles these subjobs into one parent. That is way easier to handle.